var/spack/repos/builtin/packages/sgpp/package.py

```diff
@@ -68,23 +82,39 @@ class Sgpp(SConsPackage):
         "opencl", default=False, description="Enables support for OpenCL accelerated operations"
     )
     variant("mpi", default=False, description="Enables support for MPI-distributed operations")
+    variant(
+        "eigen",
+        default=False,
+        description="Build with Eigen support",
```

Comment:
```suggestion
        description="Enables Eigen support",
```
? Just to be consistent with the descriptions above?